apps/challenges/views.py

```diff
@@ -3320,7 +3323,10 @@ def create_or_update_github_challenge(request, challenge_host_team_pk):
                 challenge_phase = ChallengePhase.objects.filter(
                     challenge__pk=challenge.pk, config_id=data["id"]
                 ).first()
-                if challenge_test_annotation_file:
+                if (
+                    challenge_test_annotation_file
+                    and challenge_phase.annotations_uploaded_using_cli is False
```

Comment:
@savish28 this check will stop updating the complete challenge phase model object after annotations are uploaded using cli once right? We only want to stop overwriting annotations

Reply:
Hey, this check would update the whole challenge phase model because annotations_uploaded_using_cli is False meaning that cli has not been used to update annotations. So it is a complete update, but when annotations_uploaded_using_cli is True we want a partial update and that is done in elif statement that would stop overwriting annotations.

Reply:
@savish28 can we change `challenge_phase.annotations_uploaded_using_cli is False` to `not challenge_phase.annotations_uploaded_using_cli`